server/templates/staff/navbar.html

```diff
@@ -42,7 +42,11 @@
 <nav class="navbar navbar-static-top" role="navigation">
   <div class="container">
     <div class="navbar-header">
-      <a href="{{ url_for('admin.index') }}" class="navbar-brand">
+      {% if current_user.is_admin %}
+        <a href="{{ url_for('admin.index') }}" class="navbar-brand">
+      {% else %}
+        <a href="{{ url_for('student.index') }}" class="navbar-brand">
+      {% endif %}
```

Comment:
If the current user is an admin, isn't the url for student.index the same as the url for admin.index? In other words, isn't this entire code block the same as the following? 

```
<a href="{{ url_for('student.index') }}" class="navbar-brand">
```

Reply:
I don't think so. Those are two entirely different routes.

Reply:
I'm running a local server and when I impersonate a random non-admin user, I can view both 
http://localhost:5000/ and http://localhost:5000/admin/course/. But when I'm impersonating an admin, http://localhost:5000/ (aka student.index) seems to redirect to http://localhost:5000/admin/course/ (admin.index) automatically.

Reply:
Is that true even if you enroll the admin as a student in a course?

Reply:
Yes

Reply:
Ok, disregard then.

Reply:
@pbitutsky @colinschoen 
I think that replacing that if else block with link to student.index only won't do the trick
`<a href="{{ url_for('student.index') }}" class="navbar-brand">`

If only link to `student.index` would stay, then an user who is an admin(has **is_admin** true) by clicking on `OK` logo will be redirected to admin dashboard of a first course they have on the list. Even on the all courses view clicking logo would redirect to that one particular course and it feels weird. Little demo below. Logged in as: **chelseacurry9@berkeley.edu**

![admin redirected to one of the courses](https://user-images.githubusercontent.com/16419971/51350820-79657c00-1aa9-11e9-9f49-f0cf16ccf6a3.gif)

In case of those users who are also a staff for a course, they will get to the `student.index` but staff edition (staff dashboard). Logged in as: **ctf3@gmail.com**

![staff_logged](https://user-images.githubusercontent.com/16419971/51351444-2d1b3b80-1aab-11e9-9d30-35d06ba0658c.gif)

And finally normal student (not a staff for any course) will always get redirected to the `student.index` and is not able to access admin dashboard
Logged in as: **gzg@gmail.com**

![image](https://user-images.githubusercontent.com/16419971/51351815-25a86200-1aac-11e9-885c-5d982f941471.png)

Please, let me know if I understood it correctly and if any more changes are neccessary :)

Reply:
In that case, this PR LGTM.